code/student/user.py

```diff
@@ -32,35 +32,43 @@ def __init__(self, uid, name, email, linkedin, degree, majors, minors, gpa, year
         self.work_auth: work_auth
         self.sponsor: sponsor
 
-client = boto3.client('dynamodb', region_name=os.environ.get('AWS_REGION', 'us-east-2'))
-dynamo_table = 'infra-resume-book-users'
+client = boto3.client('dynamodb', region_name='us-east-2')
+dynamo_table = "infra-resume-book-users"
 
 def get_user(id: int) -> str | None:
     response = client.get_item(
         TableName=dynamo_table,
         Key={
-            'id': id
+            "uin": id
```

Comment:
If this is university UIN, we are not permitted to ask for this number (and we shouldn't have it anyway).

Reply:
this refers to object_id in aad